.gitignore

```diff
@@ -1,5 +1,7 @@
 /bazel-*
 
+**/.rules_xcodeproj
```

Comment:
I'll need to add something to the FAQ or elsewhere talking about what this directory is, so people can ignore it just like this.